Prototyping/Kayla-Camera/webApp/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "miladmessage",
```

Comment:
please change these fields and regenerate package-lock

Reply:
Nice catch!  Will do.